crates/bevy_render/src/camera/bundle.rs

```diff
@@ -93,11 +93,16 @@ impl OrthographicCameraBundle {
     /// corresponding to `Z=+999.9` (closest to camera) to `Z=-0.1` (furthest away from
     /// camera) in world space.
     pub fn new_2d() -> Self {
+        Self::custom_2d(1000.0, 1.0)
+    }
+
+    /// Create an orthographic projection camera with a custom Z position and scale.
```

Comment:
Might want to add some documentation indicating that the camera will be placed at a z-position of `far - 0.1` looking toward the world origin (like in the documentation for `new_2d()`).